content/content-zh-TW.md

```diff
@@ -457,4 +457,4 @@ Blue1Brown 制作的 YouTube 影片，從音頻的角度解釋傅立葉轉換的
 
 ## 譯者
 
-您好，我是培鈞！我畢業於正修科技大學電機工程所，目前開了一間工作室 [金昶電子行](https://jinchang-electro.com) 專門協助中小企業，以及學生們開發一些小專案。如果有相關的知識或技術可以分享或研討，歡迎透過email <span id="email-mark-text">mark [at] jinchang-electro [dot] com</span>發送電子郵件給我。
```

Comment:
Would you like me to make it so that the JavaScript replaces the [at] and [dot] with @ and .? Because it runs with JavaScript instead of giving the email as plain text it can reduce the email spam you'll receive.

Reply:
I think it's fine. My firewall seems to be able to filter some spam.